contrib/virt-manager/template.py

```diff
@@ -0,0 +1,104 @@
+pkgname = "virt-manager"
+pkgver = "4.1.0"
+pkgrel = 0
+build_style = "python_pep517"
+_deps = [
+    "libxml2-python",
+    "libosinfo",
+    "python-libvirt",
+    "python-gobject",
+    "python-requests",
+    "qemu-img",
+]
+hostmakedepends = [
+    "gettext",
+    "gtk-update-icon-cache",
+    "python-build",
+    "python-docutils",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+depends = [
+    "gtk-vnc",
+    "gtksourceview4",
+    "libvirt-glib",
+    "spice-gtk",
+    "virt-manager-tools",
+    "vte-gtk3",
+]
+_checkdepends = [
+    "python-pytest",
+    "xorriso",
+]
+checkdepends = _checkdepends + _deps
```

Comment:
Drop `_checkdepends`  and use `checkdepends = ["python-pytest", "xorriso"] + _deps`